sqlserver/datadog_checks/sqlserver/schemas.py

```diff
@@ -0,0 +1,366 @@
+                 }
+        for db, schemas_by_id in self.db_to_schemas.items():
+            db_info = {}
+            if db not in self.db_info:
+                #TODO log error
+                db_info["name"] = db
+            else:
+                db_info = self.db_info[db]
+            event["metadata"] =  event["metadata"] + [{**(self.tmp_modify_to_fit_in_postgres(db_info)), "schemas": list(schemas_by_id.values())}]
+        #TODO Remove Debug Code, calculate tables and schemas sent : 
+        schemas_debug = list(schemas_by_id.values())
+        t_count = 0
+        printed_first = False
+        for schema in schemas_debug:
+            t_count += len(schema['tables'])
+            if not printed_first and len(schema['tables']) >0:
+                printed_first = True
+                self._log.warning("One of tables db {} schema {} table {}".format( list(self.db_to_schemas.keys()), schema['name'], schema['tables'][0]["name"]))
+
+        self._log.warning("Boris Adding event to Agent queue with : {} schemas and {} tables.".format(len(schemas_debug), t_count))
+        #END debug code
+        json_event = json.dumps(event, default=default_json_event_encoding)
+        self._log.debug("Reporting the following payload for schema collection: {}".format(json_event))
+        self._submit_to_agent_queue(json_event)
+        self.db_to_schemas = {}
+
+#TODO Introduce total max for data
+class Schemas:
+    def __init__(self, check):
+        self._check = check 
+        self._log = check.log
+        self._tags = [t for t in check.tags if not t.startswith('dd.internal')]
+        self._tags.append("boris:data")
+        self.schemas_per_db = {} 
+        """
+        base_event = {
+                "host": self._check.resolved_hostname,
+                "agent_version": datadog_agent.get_version(),
+                "dbms": "sqlserver", #TODO ?
+                "kind": "", # TODO 
+                #"collection_interval": self.schemas_collection_interval,
+                #"dbms_version": self._payload_pg_version(),
+                #"tags": self._tags_no_db,
+                #"cloud_metadata": self._config.cloud_metadata,
+            }
+        """
+        #TODO error is just so that payload passes, shoud be removed
+        hostname = "error"
+        if self._check.resolved_hostname is not None:
+            hostname = self._check.resolved_hostname
+        base_event = {
+            "host": hostname,
+            "agent_version": datadog_agent.get_version(),
+            "dbms": "postgres", #TODO fake it until you make it - trying to pass this data as postgres for now
+            "kind": "pg_databases", # TODO pg_databases - will result in KindPgDatabases and so processor would thing its postgres 
+            "collection_interval": 0.5, #dummy
+            "dbms_version": "v14.2", #dummy but may be format i v11 is important ?
+            "tags": self._tags, #in postgres it's no DB.
```

Comment:
make sure youre not passing internal only tags https://github.com/DataDog/integrations-core/blob/master/sqlserver/datadog_checks/sqlserver/sqlserver.py#L147

or in other places we do this https://github.com/DataDog/integrations-core/blob/master/sqlserver/datadog_checks/sqlserver/statements.py#L206

Reply:
thx, will take a look